vsintegration/src/vs/FsPkgs/FSharp.VS.FSI/fsiSessionToolWindow.fs

```diff
@@ -587,13 +595,16 @@ type internal FsiToolWindow() as this =
                  // REVIEW: log error into Trace.
                  // Example errors include no active document.
 
-    let onMLSend (sender:obj) (e:EventArgs) =       
-        sendSelectionToFSI false
+    let onMLSendSelection (sender:obj) (e:EventArgs) =       
```

Comment:
The unused variable warning is enabled per-project at the moment, and it looks like it's simply missing in the VS.FSI project. That should probably be moved to common targets, i.e. `src\FSharpSource.Settings.targets`.

I think enabling the warning and handling the required cleanup can be done in a separate PR.